File: tests/fixtures/php/else_if_chain.php
<?php
// Expected metrics:
// classify: cognitive=TBD, cyclomatic=TBD, sloc=TBD
function classify($x) {
    if ($x > 100) {
        return "high";
    } elseif ($x > 50) {
        return "medium";
    } elseif ($x > 0) {
        return "low";
    } else {
        return "negative";
    }
}

File: tests/fixtures/kotlin/else_if_chain.kt
fun classify(x: Int): String {
    if (x > 100) {
        return "big"
    } else if (x > 50) {
        return "medium"
    } else if (x > 0) {
        return "small"
    } else {
        return "non-positive"
    }
}
